src/app/pages/swap/swap-summary/swap-summary.tsx

```diff
@@ -58,30 +58,30 @@ export function SwapSummary() {
   return (
     <SwapSummaryLayout>
       <SwapContentLayout>
-        <Box width="100px">
-          <CheckCircleIcon />
-        </Box>
-        <Title fontSize={8} my="loose">
+        <styled.img src={WaxSeal} width="208px" height="181px" alt="All done" mt="space.07" />
+        <styled.h1 my="space.05" textStyle="heading.02">
           All done
-        </Title>
-        <Title fontSize={4}>
+        </styled.h1>
+        <styled.h2 textStyle="heading.04">
           {values.swapAmountTo} {values.swapAssetTo.balance.symbol}
-        </Title>
-        <Text fontWeight={500} mb="loose">
+        </styled.h2>
+        <styled.h3 mb="space.05" textStyle="label.02">
           ~{amountAsFiat}
-        </Text>
+        </styled.h3>
         <SwapAssetsPair />
-        <SwapSummaryTabs details={<SwapDetails />} status={<SwapStatus />} />
+        <SwapSummaryTabs>
+          <Outlet />
+        </SwapSummaryTabs>
       </SwapContentLayout>
       <SwapFooterLayout>
-        <Stack isInline spacing="base" width="100%">
+        <HStack gap="space.04" width="100%">
```

Comment:
Nice work here, I was trying to figure out how to replace `<Stack isInline`, I'll use `<HStack` 👍